base/asyncmap.jl

```diff
@@ -22,17 +23,17 @@ end
 
 function AsyncCollector(f, results, c...; ntasks=0)
     if ntasks == 0
-        ntasks = 100
+        ntasks = max(nworkers(), 100)
```

Comment:
thanks amit.  the problem with 100 workers is not with `pmap`, but rather that i sometimes get errors when trying to `addprocs(N)`, when N is large (say 100).  based an a comment (by @tkelman ?), i initially thought it might be due to all the tasks used by the master process during the launch, but now i think its a TCP/IP communication problem.  the error is:

```
Error [connect: address not available (EADDRNOTAVAIL)] on 42 while connecting to peer 8. Exiting.
Worker 42 terminated.
ERROR (unhandled task failure): EOFError: read end of file
```

can you suggest how i might debug this?